zotero2readwise/zt2rw.py

```
@@ -56,6 +114,18 @@ def get_all_zotero_items(self) -> list[dict]:
         return items
 
     def run(self, zot_annots_notes: list[dict] = None) -> None:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)